### ISSUE TITLE
accepting new members?

### ISSUE THREAD

**Issue by emotronica** [AnarchoTechNYC/meta#40]
hello hello! 
don't know how to really introduce myself but in short i came across this tech collective and am really interested in joining! if there's any way to become a member, please let me know. :)

**Comment by fabacab** [comment]
Hi @emotronica.

"Joining" has a somewhat loose definition, as you can imagine. [Traditionally, "joining" just means showing up](https://github.com/AnarchoTechNYC/meta/wiki/Guides#welcome-guides) at events where others already involved are hanging out or doing work together, and then contributing meaningfully and constructively while you're there. Since COVID-19 shut down a huge swath of in-person events, this is now largely a word-of-mouth situation involving [Signal Private Messenger](https://github.com/AnarchoTechNYC/meta/wiki/Signal-Private-Messenger) threads, online interactions (just like this one), and so on.

Practically speaking, this means "membership" is simply a function of how strong your friendship(s) are with the people who are already involved. Although there are far fewer in-person events now than there were last year, there are still many opportunities for you to involve yourself. The more you do so, and the more we get to know you in the process, the more of a "member" you become.

Opening an issue asking this question is a good example of one way to start getting involved. Next steps might be:

* Opening a new issue and suggesting an improvement/pointing out an error that you've spotted somewhere in our existing written materials.
* Forking any part of our codebase, adding to it, and then submitting a GitHub Pull Request for us to review and possibly merge back into our own codebase.
* Seeking out and participating in any of our own or affiliated events, which you can find from any of the [calendars listed on our wiki](https://github.com/AnarchoTechNYC/meta/wiki/Calendars).
* Finding ways to get involved with other groups with whom we sometimes work. (See [this page](https://github.com/AnarchoTechNYC/meta/wiki/Accomplices-and-allies).) This option may actually eventually lead you back here, who knows? :)

If you're familiar with the concept of Free, Libre, and Open Source Software, the above will no doubt be familiar to you, because it turns out the two concepts (FLOSS communities and Anarchist communities) share an enormous overlap in terms of methods, philosophies, and practices. It's a shame more free and open source software advocates don't recognize the similarities, or there would probably be more self-described Anarchists than there are on GitHub, but what ultimately matters is not the words one uses to describe oneself. It's the actions one takes to define oneself.

In other words: welcome! Have a look around. We're excited to see how you'll choose to contribute.
